scrutiny_vs_scrapy_comparison/README.md

```diff
@@ -0,0 +1,18 @@
+# Scrutiny vs. Scrapy Comparison Info
```

Comment:
Have we agreed on carrying over all this legacy work into this new/clean repo?  It seems unneeded here?  I'm thinking in terms of "Why would we want all this deployed to a production environment?"